lib/utils.js

```diff
@@ -120,18 +125,61 @@ const getNameString = (files, itemName, regex) => {
   return nameString;
 };
 
+/**
+ * This function returns a promise that resolves after a particular time.
+ * @param {Number} ms - Sleep time.
+ */
+const sleep = (ms) => {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+};
+
 /**
  * This function is used to obtain the project owner from a changelog label.
  *
  * @param {String} changelogLabel
  */
 const getProjectOwnerFromLabel = (changelogLabel) => {
-  const usernameDefiner = '@'
-  const labelSubstrings = changelogLabel.split(usernameDefiner);
+  const USERNAME_IDENTIFIER = '@';
+  const labelSubstrings = changelogLabel.split(USERNAME_IDENTIFIER);
   const projectOwner = labelSubstrings[labelSubstrings.length - 1].trim();
   return projectOwner;
 };
 
+/**
+ * This function gets the changelog label from a pull request.
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
+const getChangelogLabelFromPullRequest = (pullRequest) => {
+  const label = pullRequest.labels.find((label) =>
+    label.name.toUpperCase().startsWith('PR CHANGELOG')
+  );
+  if (label) {
+    return label.name.trim();
+  }
+};
+
+/**
+ * This function checks if the project owner can be assigned to a PR.
+ *
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ * @param {String} changelogLabel
+ */
+const canAssignProjectOwner = (pullRequest, changelogLabel) => {
```

Comment:
Is this function being used anywhere else other than check pull req label module?

Reply:
No, Removed.